cmd/vpp-agent-ctl/static_routing.sh

```diff
@@ -16,17 +16,13 @@ RPD_NAME="rpd"
 #
 
 # VSWITCH - add static route to 112.1.1.3/32 via 8.42.0.1
-vpp-agent-ctl -put /vnf-agent/${VSWITCH_NAME}/vpp/config/v1/vrf/0/fib - << EOF
+vpp-agent-ctl -put /vnf-agent/${VSWITCH_NAME}/vpp/config/v1/vrf/0/fib/112.1.1.3/32 - << EOF
```

Comment:
Hmmm, I am thinking:
- if we would be strictly REST URI oriented then the last character "/" would be some escape string
- on the other hand it is really good readable